apps/pipe-easy/src/app/app.component.ts

```diff
@@ -1,21 +1,36 @@
 import { NgFor } from '@angular/common';
-import { Component } from '@angular/core';
+import {
+  ChangeDetectorRef,
+  Component,
+  Pipe,
+  PipeTransform,
+} from '@angular/core';
+
+@Pipe({
+  standalone: true,
+  pure: true,
+  name: 'heavyComputation',
+})
+export class HeavyComputationPipe implements PipeTransform {
+  transform(value: string, ...args: [index: number]) {
```

Comment:
Why did you write `...args: [index: number]` this way ? 

and not only `index: number`

Reply:
You are right, that can be simplified.
I was already thinking about how to solve the typed challenge 😃